packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -699,6 +701,18 @@ suite('lit-html', () => {
       ]);
     });
 
+    test('renders to multiple attribute expressions without quotes', () => {
+      render(
+        html`<div foo=${'Foo'} bar=${'Bar'} baz=${'Baz'}></div>`,
```

Comment:
That doesn't work in 1.x. Here is `<div foo=${'Foo'}bar=${'Bar'}></div>` renders `<div foo="Foobar=Bar">`, which is arguably correct. The other arguably correct option is `<div foo="Foo" bar="Bar">`.

What's interesting is that `<div foo=${'Foo'}${'Bar'}></div>` now renders to `<div foo="FooBar"></div>`.

Do we want to officially support that?

Reply:
Yeah, I think we shouldn't sort of infer "whitespace" semantics of adding an expression (since the user can and almost always will add it themselves if that's what they intend), and so the current behavior seems both correct and good to me for both of those cases.

Reply:
Ok. It locks us into the pretty sophisticated scanner, but I guess we're committed to that. Added a test.